tests/tamr_client/test_operation.py

```diff
@@ -110,3 +110,22 @@ def test_operation_poll():
     assert op2.url == op1.url
     assert not tc.operation.succeeded(op1)
     assert tc.operation.succeeded(op2)
+
+
+def test_operation_check_success():
+    s = fake.session()
+    url = tc.URL(path="operations/1")
+    op_json = utils.load_json("operation_succeeded.json")
+    op = tc.operation._from_json(url, op_json)
+
+    tc.operation.check(s, op)
+
+
+def test_operation_failed_success():
+    s = fake.session()
+    url = tc.URL(path="operations/1")
+    op_json = utils.load_json("operation_failed.json")
+    op = tc.operation._from_json(url, op_json)
+
+    with pytest.raises(tc.operation.Failed):
+        tc.operation.check(s, op)
```

Comment:
Since part of this functionality includes passing a meaningful error message, is there a way to test that the status returned by server is surfaced correctly here?

Reply:
Done